suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -32,9 +33,10 @@ type RenderTransactionItemParams = {
     section: { monthKey: MonthKey; data: WalletAccountTransaction[] };
     index: number;
     accountKey: AccountKey;
+    tokensAreIncluded: boolean;
```

Comment:
I would prefer name `areTokensIncluded` as boolean variables should start with prefix is/are/should 

please fix this everywhere

Reply:
fixed in 0bd164f80